ui/imports/shared/controls/AccountSelector.qml

```diff
@@ -0,0 +1,144 @@
+import QtQuick 2.15
```

Comment:
I would also consider leaving `AccountSelector` without specific `contentItem` and states for styling. And then derive two selectors customizing `contentItem`. Then there is no need for states for style and Style enum.

![Screenshot from 2024-06-10 18-16-08](https://github.com/status-im/status-desktop/assets/20650004/eae64a7c-c7f7-4ac7-bae9-e89fe3af4da4)

Reply:
That's true, good point!